src/typeDefs/errors/mutationErrors.ts

```diff
@@ -0,0 +1,20 @@
+import { gql } from "graphql-tag";
+
+export const mutationErrrors = gql`
+  union AcceptAdminError = CurrentUserNotFound | GivenUserNotFound
+
+  interface UserNotFound {
+    message: String!
+    path: [String!]!
+  }
+
+  type CurrentUserNotFound implements UserNotFound {
```

Comment:
I believe the intention here was to define an error type when user is unauthenticated? If so that should be covered with a more specific type like `UnauthenticatedError`.